app/views/articles/_sidebar_additional.html.erb

```diff
@@ -3,11 +3,31 @@
   <aside class="side-bar sidebar-additional showing grid gap-4" id="sidebar-additional">
     <% unless user_signed_in? || !any_selfserve_auth? %>
       <section class="crayons-card crayons-card--secondary signin-cta-widget">
-        <header class="crayons-card__header">
-          <h3 class="crayons-card__header__headline">Join <%= community_name %></h3>
-        </header>
-        <div class="p-4 grid gap-2">
-          <%= render partial: "shared/authentication/providers_sidebar" %>
+        <div class="authentication-widget__container">
```

Comment:
I feel like at some point in the future, I could bring this code block into a partial since I reference this in multiple places.